solitude/settings/sites/paymentsalt/db.py

```diff
@@ -17,6 +17,7 @@
 DATABASES['default'] = dj_database_url.parse(private.DATABASES_DEFAULT_URL)
 DATABASES['default']['ENGINE'] = 'django.db.backends.mysql'
 DATABASES['default']['OPTIONS'] = {'init_command': 'SET storage_engine=InnoDB'}
+DATABASES['default']['ATOMIC_REQUESTS'] = True
```

Comment:
or maybe make a `base.DEFAULT_DB_SETTINGS` dict that everything can use because otherwise we'll probably forget and introduce another bug like this in the future.